src/common/text.c

```diff
@@ -368,20 +368,20 @@ const RegionName kRegionNamesOot[] = {
     { "at",         "the " TEXT_COLOR_PINK "Bottom of the Well" },
     { "in",         "the " TEXT_COLOR_TEAL "Ice Cavern" },
     { "in",         TEXT_COLOR_YELLOW "Gerudo Training Grounds" },
-    { "in",         "the " TEXT_COLOR_YELLOW "Thieve's Hideout" },
+    { "in",         "the " TEXT_COLOR_YELLOW "Thieves' Hideout" },
     { "inside",     TEXT_COLOR_RED "Ganon's Castle" },
     { "in",         "the " TEXT_COLOR_GREEN "Kokiri Forest" },
     { "in",         TEXT_COLOR_YELLOW "Hyrule Field" },
-    { "in",         TEXT_COLOR_YELLOW "Hyrule's Market" },
+    { "in",         TEXT_COLOR_YELLOW "Market" },
     { "in",         TEXT_COLOR_ORANGE "Lon Lon Ranch" },
-    { "in",         TEXT_COLOR_YELLOW "Hyrule's Castle" },
+    { "in",         TEXT_COLOR_YELLOW "Hyrule Castle" },
     { "in",         TEXT_COLOR_RED "Ganon's Castle Exterior" },
     { "in",         "the " TEXT_COLOR_GREEN "Lost Woods" },
-    { "in",         "the " TEXT_COLOR_GREEN "Sacred Meadow" },
+    { "in",         "the " TEXT_COLOR_GREEN "Sacred Forest Meadow" },
     { "in",         TEXT_COLOR_RED "Kakariko" },
-    { "inside",     TEXT_COLOR_PINK "Kakariko's Graveyard" },
-    { "on",         TEXT_COLOR_RED "Death Mountain's Trail" },
-    { "in",         TEXT_COLOR_RED "Death Mountain's Crater" },
+    { "inside",     TEXT_COLOR_PINK "Graveyard" },
```

Comment:
There are two graveyards (with Ikana), so it's better to name both

Reply:
I suppose that's fair, although MM's Graveyard is already demarcated by "Ikana" in front of it. But I'll just revert that back, not a big deal, just weird